src/redux/books/books.js

```diff
@@ -0,0 +1,31 @@
+export const addBook = ({ name, category }) => ({
```

Comment:
**[OPTIONAL] extra review comment **


Hello @harlexkhal  your project was selected as a random quality control check. ere is an additional comment that was missed by your code reviewer. 

You should use the dicks pattern for your reducers. Please take a look at the example here: https://github.com/erikras/ducks-modular-redux#example. You are missing 2 things, the first one:

- actions should be defined as `const` that later can be used in action creators and reducers.